packages/pages-components/src/components/link/types.ts

```diff
@@ -53,6 +53,16 @@ interface LinkConfig
   currency?: string;
   /** The monetary value of the event. */
   amount?: number;
+  /**
+   * Up to 10 pairs matching custom string keys to string values to associate with the event.
+   * Keys are case-insensitive; values are case-sensitive.
+   */
+  customTags?: Record<string, string>;
+  /**
+   * Up to 10 pairs matching custom string keys to number values to associate with the event.
+   * Keys are case-insensitive.
```

Comment:
are values also case-insensitive for customValues?

Reply:
Values are numbers. I also ripped these docs straight from the analytics lib.